vcpkg.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "mixxx",
+  "version": "2.4.0",
+  "description": "Mixxx is free cross platform DJ software",
+  "homepage": "https://mixxx.org/",
+  "license": "GPL-2.0-or-later",
+  "supports": "!uwp",
+  "dependencies": [
+    "chromaprint"
```

Comment:
Why is only `chromaprint` listed here?

Reply:
I didn't want to wait too long for the build to finish. We can add more packages when the basics are working.